sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusMessageSerializer.java

```diff
@@ -53,18 +59,9 @@
  */
 class ServiceBusMessageSerializer implements MessageSerializer {
     private static final byte[] EMPTY_BYTE_ARRAY = new byte[0];
-    private static final String ENQUEUED_TIME_UTC_NAME = "x-opt-enqueued-time";
     private static final String SCHEDULED_ENQUEUE_TIME_NAME = "x-opt-scheduled-enqueue-time";
```

Comment:
I realised this constants are already defined in: AmqpMessageConstant in azure-core-amqp.